prem/js/script.js

```diff
@@ -0,0 +1,19 @@
+let counter = 0;
+function showTable() {
+
+}
+function addData() {

```

Comment:
I'm able to add empty rows to the table. Please display some message or disable the button when no values are present

Reply:
i resolved the issue and commit the file with new changes